src/azul/plugins/repository/tdr/__init__.py

```diff
@@ -452,7 +455,7 @@ def _merge_links(self, links_jsons: JSONs) -> JSON:
             merged_content['links'] = sum((sc['links'] for sc in source_contents),
                                           start=[])
             merged['content'] = merged_content  # Keep result of parsed JSON for reuse
-            merged['content_size'] = len(json.dumps(merged_content))
+            merged['content_size'] = json_compact_strlen(merged_content)
```

Comment:
does the JSON value in merged['content'] have minimal white spacing? I don't see any additional modifications, unless this is addressing a bug in the code...
Should we have a test that asserts that the bytes in the bundle content match the reported bytes from this field?

Reply:
`merged['content']` is a a dictionary, not a JSON-formatted string, so the concept of "whitespace" does not apply to it. The changes to `content-size` for the other entities is for accuracy, but this one is for consistency since there is no corresponding JSON document in TDR.

Reply:
I think a test to assert content size would be redundant since we'd be running the same code to calculate the size and verify it. I think the doctests for the functions in `src/azul.json.py` are sufficient.